pkg/api/server/config/config.go

```diff
@@ -29,6 +29,9 @@ type Config struct {
 	LOGS_BUFFER_SIZE int    `mapstructure:"LOGS_BUFFER_SIZE"`
 	LOGS_PATH        string `mapstructure:"LOGS_PATH"`
 
+	GCS_BUCKET_NAME       string `mapstructure:"GCS_BUCKET_NAME"`
+	STORAGE_EMULATOR_HOST string `mapstructure:"STORAGE_EMULATOR_HOST"`
```

Comment:
Is this intended to be GCS specific? If so, please prefix this config item with `GCS`:

```suggestion
	GCS_STORAGE_EMULATOR_HOST string `mapstructure:"GCS_STORAGE_EMULATOR_HOST"`
```

Reply:
GCS driver uses this exact value.  It uses the `STORAGE_EMULATOR_HOST` environment variable.

Reply:
Please add a comment above with this information.